ts/src/pro/ascendex.ts

```diff
@@ -218,7 +218,7 @@ export default class ascendex extends ascendexRest {
          */
         await this.loadMarkets ();
         const market = this.market (symbol);
-        const channel = 'depth-realtime' + ':' + market['id'];
```

Comment:
why are we updating this?

Reply:
there were not `depth-realtime` channels (at least, note anymore, i cant find them in api docs).

Reply:
but isn't the master version working?
```
 n ascendex watchOrderBook "BTC/USDT"       
Debugger attached.
2024-02-20T12:09:57.887Z
Node.js: v18.18.0
CCXT v4.2.48
ascendex.watchOrderBook (BTC/USDT)
{
  bids: [
    [ 52313.46, 0.00997 ], [ 52217, 0.0015 ],     [ 52173.2, 0.216 ],
    [ 52164.31, 0.216 ],   [ 52140.5, 0.422 ],    [ 52107.8, 0.422 ],
  ],
  asks: [
    [ 53497, 0.00195 ],    [ 53500, 0.00467 ],    [ 53509.74, 0.003 ],
    [ 92000, 0.01 ],       [ 100000, 0.00005 ]
  ],
  timestamp: 1708431010545,
  datetime: '2024-02-20T12:10:10.545Z',
  nonce: 8263423,
  symbol: 'BTC/USDT'
}
```

Reply:
across api docs https://ascendex.github.io/ascendex-futures-pro-api-v2/ or spot, there is no mention of "realtime" phrase anywhere. i suppose it was the old version. atm, the incoming message still mentions it: 
```
{
  m: 'depth-realtime',
  symbol: 'BTC/USDT',
  data: { .... }
}
```
and the `handleOrderBook` handler is attached to that message, yes, that's why it works. however, it's obsolete and we should not subscribe to `depth-realtime` (even though exchange seems to support for backward-compatibility, but probably will be removed, as the other subscription with `depth-snapshot-realtime` is not being recognized & subscribe by exchange anymore).
so, nowhere in subscription/request we need to use `realtime`, everything works without it well and is compliant with apidocs

Reply:
but during merge that issue might not happen anymore, because that `bid == ask` was intermittent probably and will go away

Reply:
The thing is the current implementation does not require us to fetch the snapshot separately right?

Reply:
current implementation does not work for swaps , that's why it didn't have it

Reply:
in spot docs, there is `depth-snapshot` while in contracts they don't have `depth-snapshot`, however, whenever ascendex WS was implemented in the past, the dev hadn't addressed it for contract markets